astropy/modeling/fitting.py

```diff
@@ -411,9 +411,7 @@ def _is_invertible(m):
         """Check if inverse of matrix can be obtained."""
         if m.shape[0] != m.shape[1]:
             return False
-        if np.linalg.matrix_rank(m) < m.shape[0]:
-            return False
-        return True
+        return not np.linalg.matrix_rank(m) < m.shape[0]
```

Comment:
```suggestion
        return np.linalg.matrix_rank(m) >= m.shape[0]
```
If there is some `nan` weirdness going on then it should be mentioned in a comment or in the docstring.